ext/native-decls/GetVehicleDoorStatus.md

```diff
@@ -5,12 +5,14 @@ apiset: server
 ## GET_VEHICLE_DOOR_STATUS
 
 ```c
-int GET_VEHICLE_DOOR_STATUS(Vehicle vehicle);
+int GET_VEHICLE_DOOR_STATUS(Vehicle vehicle, int doorIndex);
```

Comment:
```suggestion
int GET_VEHICLE_DOOR_STATUS(Vehicle vehicle, cs_split int doorIndex);
```

We need to split the declaration so we don't break ABI compatibility with C# (even though this wouldn't have worked anyways.

Reply:
Good catch, thx!